test/test-Worker.js

```diff
@@ -3,6 +3,7 @@ const { toBeType } = require('jest-tobetype');
 const mediasoup = require('../');
 const { createWorker, observer } = mediasoup;
 const { InvalidStateError } = require('../lib/errors');
+const os = require('os');
```

Comment:
Let's move system/core modules to the top, this, to the first line of this file.

Reply:
agree.